pkg/build-deb.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
```

Comment:
Needed to change this to `#!/bin/bash` to get the script to work with Ubuntu 18.04.3.